### PR TITLE
 Mass and rinertia tests refactoring

### LINKED ISSUE #1905: Thermo test readability improvement
I will refactor the test mass-and-rinertia_per_particle.py in order to make it more readable according to advices of @KaiSzuttor.

### PR DESCRIPTION
**Pull request by bogdan-tanygin**
Fixes #1905
Test refactoring according to [advice](https://github.com/espressomd/espresso/commit/5ba2c41a3d8e391d072f13bc1f71f6c706422f49#r28083156)/[advice](https://github.com/espressomd/espresso/pull/1757#issuecomment-392322098) from @KaiSzuttor and @RudolfWeeber 
Description of changes:
 - Refactored the naming and comments
 - The test cases loop has been refactored into the separate test cases
 - A code duplication is reduced
 - A code size is reduced by a removing of unnecessary type conversions
 - More advanced `unittest` usage

PR Checklist
------------
 - [ ] Tests?
   - [ ] Interface
   - [ ] Core 
 - [ ] Docs?

### CODE REVIEW

File: testsuite/mass-and-rinertia_per_particle.py [hunk 1]
```diff
@@ -291,52 +211,28 @@ def run_test_case(self, test_case):
                     # integration of the Langevin EOM
                     sigma2_tr[k] = 0.0
                     for j in range(3):
-                        sigma2_tr[k] = sigma2_tr[k] + D_tr[k,
+                        sigma2_tr[k] += self.D_tran_p_validate[k,
```
Comment (KaiSzuttor) [inline testsuite/mass-and-rinertia_per_particle.py]: could you please add a reference for the formula you compare to

File: testsuite/mass-and-rinertia_per_particle.py [hunk 2]
```diff
@@ -14,256 +14,176 @@
...
                     if "ROTATION" in espressomd.features():
                         self.assertLess(abs(
-                            self.es.part[k].omega_body[j] - math.exp(- gamma_rot_validate[k, j] * self.es.time / J[j])), tol)
+                            self.es.part[k].omega_body[j] - math.exp(- self.gamma_rot_p_validate[k, j] * self.es.time / self.J[j])), tol)
 
-        for i in range(len(self.es.part)):
-            self.es.part[i].remove()
-
-        # thermalization
+    def fluctuation_dissipation(self,n):
```
Comment (KaiSzuttor) [inline testsuite/mass-and-rinertia_per_particle.py]: could you please name the methods according to what they do, like ```check_fluctuation_dissipation``` or ```set_xy_variable```

File: testsuite/mass-and-rinertia_per_particle.py [hunk 3]
```diff
@@ -14,256 +14,176 @@
...
-        if test_case == 1 or test_case == 3:
-            gamma_tr = gamma_tran
-            gamma_rot_validate = gamma_rot
-        else:
+            self.gamma_tran_p[k, :] = (0.4 + np.random.random(3)) * 10
+            self.gamma_rot_p[k, :] = (0.2 + np.random.random(3)) * 20
+        
+        ## Particles
+        self.mass = (0.2 + np.random.random()) * 7.0
+        self.J = (0.2 + np.random.random(3)) * 7.0
```
Comment (KaiSzuttor) [inline testsuite/mass-and-rinertia_per_particle.py]: where do the magic ```0.2``` and ```0.7``` come from? In general magic number should have a descriptive name

File: testsuite/mass-and-rinertia_per_particle.py [hunk 4]
```diff
@@ -14,256 +14,176 @@
...
-        # gamma_tran/gamma_rot matrix: [2 types of particless] x [3 dimensions
-        # X Y Z]
-        gamma_tran = np.zeros((2, 3))
-        gamma_tr = np.zeros((2, 3))
-        gamma_rot = np.zeros((2, 3))
-        gamma_rot_validate = np.zeros((2, 3))
-        # The translational gamma isotropy is required here. See comments below.
+        print("\n")
+
+    def langevin_global_defaults(self):
```
Comment (KaiSzuttor) [inline testsuite/mass-and-rinertia_per_particle.py]: also here: please name methods according to what they do

File: testsuite/mass-and-rinertia_per_particle.py [hunk 5]
```diff
@@ -14,256 +14,176 @@
...
-                self.es.part[i].omega_body = np.array([1.0, 1.0, 1.0])
-            self.es.part[i].mass = mass
-            self.es.part[i].rinertia = np.array(J)
-
-        print("\n")
+                self.es.part[i].omega_body = 1.0,1.0,1.0
+            self.es.part[i].mass = self.mass
+            self.es.part[i].rinertia = self.J
 
+    def fluctuation_dissipation_param_setup(self,n):
```
Comment (KaiSzuttor) [inline testsuite/mass-and-rinertia_per_particle.py]: could you please use the numpy-style in-code documentation that is used for python functions in espresso and describe what the argument to the function means and what type it has

File: testsuite/mass-and-rinertia_per_particle.py [hunk 6]
```diff
@@ -364,14 +260,172 @@ def run_test_case(self, test_case):
...
+                ind = i + k * n
+                self.es.part[ind].temp = self.kT_p[k]
+
+    def diffusivity_tran_assign(self):
+        for k in range(2):
+            # Translational diffusivity for a validation
+            self.D_tran_p_validate[k, :] = 2.0 * self.halfkT_p_validate[k] / self.gamma_tran_p_validate[k, :]
+
+    def test_case_00(self):
+        print("-------------------------------------------------------------")
```
Comment (KaiSzuttor) [inline testsuite/mass-and-rinertia_per_particle.py]: Please avoid print statements in testcases.

**Review by KaiSzuttor — CHANGES_REQUESTED** [review]
Thanks alot @psci2195, it's a lot more readable now! To make me even more happy, could you please address my comments?

**Comment by bogdan-tanygin** [comment]
@KaiSzuttor thanks for a useful review. I've implemented changes according to each bullet raised by you.

**Review by KaiSzuttor — APPROVED** [review]

**Pull request merged**